mmeds/util.py

```diff
@@ -944,13 +946,17 @@ def quote_sql(sql, quote='`', **kwargs):
     return formatted
 
 
-def make_pheniqs_config(reads_forward, reads_reverse, barcodes_forward, barcodes_reverse, mapping_file, o_directory):
+def make_pheniqs_config(reads_forward, reads_reverse, barcodes_forward, barcodes_reverse, mapping_file, o_directory,
```

Comment:
Not gonna force you to write more tests here, but the `util.py` changes are what's causing the codecov to fail. Are there any tests for these new functions?